src/06-loops.js

```diff
@@ -25,7 +31,14 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
+function rangeOdd(min, max) {
+     let arr =[ ];
+  for (let i = max; i >= min; i--) {// Reversal
```

Comment:
I would suggest not iterating backwards if we can avoid it.  Most developers will assume we iterate forwards.  Also - can we change how we iterate if we are only looking for odd ranges?